compositor_render/src/transformations/image/svg_image.rs

```diff
@@ -77,7 +72,7 @@ impl SvgAsset {
             }
             (SvgRenderer::CpuOptimized, NodeTextureState::CpuOptimized { texture, .. }) => {
                 // input is already in sRGB with pre-multiplied alpha
-                render_to_texture(ctx, &self.tree.0, texture.texture(), resolution);
+                render_to_texture(ctx, &self.tree.0, texture.texture(), state.resolution());
             }
             (SvgRenderer::WebGl(renderer), NodeTextureState::WebGl { texture, .. }) => {
                 renderer.render(ctx, &self.tree.0, texture, resolution)
```

Comment:
You're using `state.resolution()` only for cpu optimized rendering but for other rendering types you rely on `SvgAsset::resolution`. Was that intentional?